src/message/LogMessage.hpp

```diff
@@ -38,6 +38,7 @@ namespace message {
         std::string message;
 
         /// @brief The currently executing task that made this message
+        /// @warning This can point to garbage if this message is not acted on instantly.
```

Comment:
```suggestion
        /// @warning This pointer is only valid for the duration of the reaction that triggers on this message.
        ///          After this, it will point to junk memory. This also applies to keywords that reschedule this reaction
        ///          For example, using the Sync keyword can make it so this pointer is invalid
```